client/utils/activity/titles/user.ts

```diff
@@ -27,13 +26,7 @@ const getUserTitleFromUserIdAndContext = (
 };
 
 export const actorTitle: TitleRenderer<InsertableActivityItem> = (item, context) => {
-	const { otherActorsCount } = context;
```

Comment:
Please ignore these last few files here — these are a result of a mistake when splitting up a branch into many PRs.